manning/manning-proposal-2020-07.md

```diff
@@ -203,24 +197,23 @@ Name (as it would appear on a book cover):
 -   Charlotte Wickham
 -   Greg Wilson
 
-Website/blogs/Twitter, etc:
+### Social media
 
 -   Damien Irving: https://damienirving.github.io/
 -   Kate Hertweck: http://katehertweck.com/
 -   Luke Johnston: http://lukewjohnston.com/
--   Luke Johnston: https://scholar.google.ca/citations?user=8zCoa-UAAAAJ
+-   Sara Mahallati: https://scholar.google.ca/citations?user=8zCoa-UAAAAJ
 -   Joel Ostblom: https://joelostblom.com/
 -   Charlotte Wickham: https://www.cwick.co.nz/
 -   Greg Wilson: http://third-bit.com/
 
-**9. Schedule**
+## 9. Schedule
 
 First draft complete.
 
-**10. Table of Contents**
-
-FIXME: UPDATE PROJECTS AND PUBLISHING CONTENTS.
+## 10. Table of Contents
```

Comment:
The table of contents might need to change a bit if we adopt the proposal in #415.

Reply:
Yup.